docs/_docs/installation.md

```diff
@@ -37,11 +46,34 @@ Before you start, make sure your system has the following:
   </p>
 </div>
 
+## Install & Use Jekyll with Bundler
```

Comment:
I don’t know if we want this `./vendor/bundle` method to be presented first, as if it is the default/recommended method :/